docs/README.md

```diff
@@ -7,7 +7,7 @@ Precondition: python3 and pip3 are already installed.
 ### Recommended steps:
 
 1. Install virtualenv - this allows you to create a python sandbox for your project  
-`pip3 install virtualenv`
+`pip install virtualenv`
 
 1. Create dedicated virtual environment  
 `virtualenv .venv`
```

Comment:
Use `python -m venv .venv` here.

Reply:
BTW, most developers will use virtual environment support in their IDE, `pyenv, `virtualenvwrapper` or  something like that anyway.